README.md

```diff
@@ -9,14 +9,41 @@ Users may retrieve quotes from this API based on the following:
 2. Season
 3. Episode
 
-# How to start
-```
+# Getting started
+
+### Clone the repository
+You can clone this repository by this following command:
+``` bash
 git clone https://github.com/jeyren95/big-bang-theory-quotes
 cd big-bang-theory-quotes
-cp .env.example .env
-go run main.go
 ```
 
+### Initialize module
```

Comment:
its not necessary to build to run the application locally, some users may just want to use "go run main.go" to run the application.

can we also keep that?

Reply:
Sure.

Reply:
Should I remove `make init` instead if `build` is unnecessary?

Reply:
no, dont have to remove. Inside this README doc i think you can specify that there are 2 ways to run the project --> either build and run, or "go run main.go"

Reply:
Got it.

Reply:
I think we can remove the `build` one. So the installation steps is quite simple. Just run `make run`

Reply:
Resolved.